src/Debug.h

```diff
@@ -171,7 +171,7 @@ HERE(std::ostream& s)
 #endif
 
 /* some uint8_t do not like streaming control-chars (values 0-31, 127+) */
-inline std::ostream& operator <<(std::ostream &os, const uint8_t d)
+inline std::ostream &operator <<(std::ostream &os, const uint8_t d)
```

Comment:
> This is purely a pedantic stylistic change by me: I hope it's OK by you.

Please undo those changes. While I prefer this style as well, code style changes (of otherwise unmodified lines) are out of this PR scope. In most cases, such changes do more harm then good overall, especially when applied manually.